mbedtls-sys/build/bindgen.rs

```diff
@@ -33,7 +33,8 @@ impl super::BuildConfig {
                 Ok(for h in headers::enabled_ordered() {
```

Comment:
Try to avoid changing files without any semantic changes. If something needs to be formatted, file a separate PR for that

Reply:
Sure. It was due to rust format, but a separate PR is definitely better.